grading/load_config_json.cpp

```diff
@@ -1093,6 +1093,17 @@ void InflateTestcase(nlohmann::json &single_testcase, nlohmann::json &whole_conf
         } else {
           assert (validShowValue(*itr2));
         }
+        assert(grader.find("expected_string") == grader.end());
+      }
+      if (grader.find("expected_string") != grader.end()) {
+        itr2 = grader.find("show_expected");
+        if (itr2 == grader.end()) {
+          grader["show_expected"] = "always";
+        } else {
+          assert (validShowValue(*itr2));
+        }
+        grader["use_expected_string"] = true;
+        assert(grader.find("expected_file") == grader.end());
```

Comment:
Adding an error message here would be helpful

```suggestion
        assert(grader.find("expected_file") == grader.end() && "You cannot specify both expected_file and expected_string");
```